LiteCore/Android/unicode/ndk_icu.c

```diff
@@ -158,7 +158,7 @@ static void init_icudata_version() {
     syms[10] = dlsym(handle_i18n, func_name);
 }
 
-UCollator* ucol_open(const char* loc, UErrorCode* status) {
+UCollator* lc_ucol_open(const char* loc, UErrorCode* status) {
```

Comment:
I don't find how/where icu_shim.h is included into this file.

Reply:
It's not.  It doesn't make a difference to the build.

Reply:
yes, It simply defines the function. But with header included, the complier may help ensure some integrity.

Reply:
I don't see how it will help ensure anything, but I'll put it in anyway.